common/src/main/scala/explore/model/RootModel.scala

```diff
@@ -19,26 +19,29 @@ import lucuma.core.model.User
 import monocle.Focus
 import monocle.Lens
 import monocle.Optional
+import react.primereact.Tree
 
 import scala.collection.immutable.HashSet
 case class RootModel(
-  vault:                Option[UserVault],
-  localPreferences:     ExploreLocalPreferences,
-  expandedIds:          ExpandedIds = ExpandedIds(),
-  searchingTarget:      Set[Target.Id] = HashSet.empty,
-  userSelectionMessage: Option[NonEmptyString] = none,
-  programSummaries:     Option[ProgramSummaries] = none,
-  undoStacks:           ModelUndoStacks[IO] = ModelUndoStacks[IO]()
+  vault:                 Option[UserVault],
+  localPreferences:      ExploreLocalPreferences,
+  expandedIds:           ExpandedIds = ExpandedIds(),
+  searchingTarget:       Set[Target.Id] = HashSet.empty,
+  userSelectionMessage:  Option[NonEmptyString] = none,
+  programSummaries:      Option[ProgramSummaries] = none,
+  undoStacks:            ModelUndoStacks[IO] = ModelUndoStacks[IO](),
+  expandedObsListGroups: Map[Tree.Id, Boolean] = Map.empty
```

Comment:
Do you need a `Map` here? Or would a `Set` or `SortedSet` work to just keep track of the expanded ones?

Also, I think this should probably go in `ExpandedIds` with the other expanded states.

Reply:
The Primereact type is a Map (with the values always `true`), but I agree a `Set`is probably cleaner and we can keep the map an implementation detail for the primereact component

Reply:
Should I change it here or in lucuma-react? I don't know how close to the primereact API we want to stay in lucuma-react

Reply:
Ie. The closer we stay to the API, the easier it is to use the primereact documentation for using our components. But, if we can clean it up a bit...